odl/discr/discr_mappings.py

```diff
@@ -687,42 +790,185 @@ def _evaluate(self, indices, norm_distances, out=None):
             return self.values[idx_res]
 
 
-class _NearestMeshgridInterpolator(_MeshgridInterpolator,
-                                   _NearestPointwiseInterpolator):
+def _compute_nearest_weights_edge(idcs, ndist, variant):
+    """Helper for nearest interpolation mimicing the linear case."""
+    # Get out-of-bounds indices from the norm_distances. Negative
+    # means "too low", larger than or equal to 1 means "too high"
+    lo = np.where(ndist < 0)
```

Comment:
In an earlier version I needed the indices, too. We should be able to do without now.